### ISSUE TITLE
Add an option to always open certain links in new tabs/windows

### ISSUE THREAD

**Issue by jesperkristensen** [sorenkrabbe/Chrome-Salesforce-inspector#89]
Because some people like this.

I personally hate this, but if it is configurable, it's fine.

Need to find out if extension APIs offer a simple way to store options.

**Comment by Garywoo** [comment]
Yes, I would personally like to open the _Show all data_ action in a new tab every time. I often forget that I can ctrl+click on the link as it appears as a button in the popup menu, and button's don't usually support such actions.

**Comment by dufoli** [comment]
it is done in the fork : https://github.com/dufoli/Salesforce-Inspector-Advanced
https://chromewebstore.google.com/detail/salesforce-inspector-adva/dbfimaflmomgldabcphgolbeoamjogji

**Comment by tprouvot** [comment]
> it is done in the fork : https://github.com/dufoli/Salesforce-Inspector-Advanced https://chromewebstore.google.com/detail/salesforce-inspector-adva/dbfimaflmomgldabcphgolbeoamjogji

@dufoli Let me bring some precisions to your message : this is done in the fork [Salesforce Inspector reloaded](https://chromewebstore.google.com/detail/salesforce-inspector-relo/hpijlohoihegkfehhibggnkbjhoemldh) that you've forked.

![image](https://github.com/sorenkrabbe/Chrome-Salesforce-inspector/assets/35368290/3cdd9305-2b48-4054-8de1-6877b79d73bb)

**Comment by dufoli** [comment]
it is true, you have 2 forks currently.
reloaded and advanced. I can talk only for advanced which is forked from reloaded.
On Salesforce inspector advanced, you have sosl, new editor with better suggestion, syntax hightlight, picklist suggestion on insert, apex runner, platform event tool, profiling, flame chart... I let you check each addon and select the one you prefer.
